api.planx.uk/modules/team/service.test.ts

```diff
@@ -0,0 +1,108 @@
+import { getJWT } from "../../tests/mockJWT";
+import { userContext } from "../auth/middleware";
+
+import {
+  getUserAndTeam,
+  addMember,
+  removeMember,
+  changeMemberRole,
+} from "./service";
+
+const getStoreMock = jest.spyOn(userContext, "getStore");
+
+const mockTeam = { id: 123 };
+const mockUser = { id: 456 };
+
+const mockAddMember = jest.fn();
+const mockRemoveMember = jest.fn();
+const mockChangeMemberRole = jest.fn();
+const mockGetTeamBySlug = jest.fn().mockResolvedValue(mockTeam);
+const mockGetUserByEmail = jest.fn().mockResolvedValue(mockUser);
+
+jest.mock("@opensystemslab/planx-core", () => {
```

Comment:
At some point soon I'm going to look at how Jest can automock modules - I think this will allow us much easier setup and less repetition.